python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -221,15 +217,7 @@ def get_assets_latest_info(
 
     return [
         GrapheneAssetLatestInfo(
-            id=(
-                get_unique_asset_id(
-                    asset_key,
-                    asset_nodes[asset_key].repository_location.name,
-                    asset_nodes[asset_key].external_repository.name,
```

Comment:
verified a while back with @salazarm that we don't need this additional specificity in this ID